docs/usage.md

```diff
@@ -32,7 +32,7 @@ local function TestOneInput(buf)
     parser.parse(decoder.decode(buf))
```

Comment:
BTW `docs/api.md` should be updated too.
At least this:

> - `args` is a table with arguments: the process arguments to pass to the
>  fuzzer. Field `corpus` specifies a path to a directory with seed corpus, see a
>  list with other options in the [libFuzzer documentation][libfuzzer-options-url].

Reply:
not relevant

Reply:
Please describe passing arguments via command-line in documentation.